Jenkinsfile-nightly

```diff
@@ -16,6 +16,33 @@ pipeline {
     }
 
     stages {
+        stage('Artifact Stats') {
+            agent {
+                docker {
+                    image 'noisepage:focal'
+                }
+            }
+            steps {
+                sh 'echo $NODE_NAME'
+                sh script:'echo y | sudo ./script/installation/packages.sh all', label: 'Installing packages'
+
+                sh script:'''
```

Comment:
Minor: Can you just add a comment in here that explains what this command does and why we need it?